pyiceberg/catalog/sql.py

```diff
@@ -110,8 +110,28 @@ def __init__(self, name: str, **properties: str):
 
         if not (uri_prop := self.properties.get("uri")):
             raise NoSuchPropertyException("SQL connection URI is required")
-        echo = bool(self.properties.get("echo", False))
-        self.engine = create_engine(uri_prop, echo=echo)
+
+        echo: Union[str, bool]
+        echo_str = str(self.properties.get("echo", "false")).lower()
+        if echo_str == "debug":
+            echo = "debug"
+        elif echo_str == "true":
+            echo = True
+        elif echo_str == "false":
+            echo = False
+        else:
+            raise ValueError(f"Invalid value for echo parameter: {echo_str}")
+
+        pool_pre_ping: Union[str, bool]
+        pool_pre_ping_str = str(self.properties.get("pool_pre_ping", "false")).lower()
```

Comment:
`strtobool` might be helpful here
https://github.com/apache/iceberg-python/commit/84ab9791a65dc60a38f886193c93519b5bce5d5f

Reply:
oohh, this is like, fresh from the oven.  Cool!